sql/core/src/test/resources/sql-tests/results/ansi/string-functions.sql.out

```diff
@@ -177,8 +177,8 @@ SELECT split_part('11.12.13', '.', 0)
 -- !query schema
 struct<>
 -- !query output
-java.lang.ArrayIndexOutOfBoundsException
-SQL array indices start at 1
+org.apache.spark.SparkArrayIndexOutOfBoundsException
+[INVALID_ARRAY_INDEX_IN_ELEMENT_AT] The index 0 is out of bounds. The array has 3 elements. Use `try_element_at` to tolerate accessing element at invalid index and return NULL instead. If necessary set "spark.sql.ansi.enabled" to "false" to bypass this error.
```

Comment:
Use `try_element_at` to tolerate accessing element ... this might confuse users. The recommendation is not useful.

Reply:
Yes, i changed it.